man/systemd-stub.xml

```diff
@@ -77,7 +77,16 @@
       <listitem><para>A <literal>.splash</literal> section with an image (in the Windows
       <filename>.BMP</filename> format) to show on screen before invoking the kernel.</para></listitem>
 
-      <listitem><para>A <literal>.dtb</literal> section with a compiled binary DeviceTree.</para></listitem>
+      <listitem><para>Zero or more <literal>.dtb</literal> sections. If only one such section is present, it
+      will always be passed to the kernel. If there at least two of them, stub will first try to match the
+      DeviceTrees bundled into UKI by comparing the first <literal>compatible</literal> string in each bundled
```

Comment:
`<varname>` is probably a more suitable tag for this than `<literal>`. after all `compatible` is not a literal value used here, but the name of a field, i.e. the name of a variable